charts/kube-state-metrics/templates/role.yaml

```diff
@@ -105,6 +105,30 @@ rules:
   - networkpolicies
   verbs: ["list", "watch"]
 {{ end -}}
+{{ if has "ingressclasses" $.Values.collectors }}
+- apiGroups: ["networking.k8s.io"]
+  resources:
+  - ingressclasses
+  verbs: ["list", "watch"]
+{{ end -}}
+{{ if has "clusterrolebindings" $.Values.collectors }}
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources:
+  - clusterrolebindings
+  verbs: ["list", "watch"]
+{{ end -}}
+{{ if has "clusterroles" $.Values.collectors }}
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources:
+  - clusterroles
+  verbs: ["list", "watch"]
+{{ end -}}
+{{ if has "roles" $.Values.collectors }}
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources:
+  - roles
+  verbs: ["list", "watch"]
+{{ end -}}
```

Comment:
All collectors are enabled by default, but these will be missing.
Source: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-state-metrics/values.yaml#L371

@DrFaust92 Could you also add them to the values file?

Reply:
@dotdc they are also not enabled by default in upstream https://github.com/kubernetes/kube-state-metrics/blob/main/pkg/options/resource.go#L28

Reply:
Maybe add them but commented out, wdyt ?
Otherwise it's fine to merge as is. 👍

Reply:
sure

Reply:
@dotdc added